Src/dronecan_application/modules/CircuitStatusModule.hpp

```diff
@@ -0,0 +1,53 @@
+/***
+ * Copyright (C) 2024 Anastasiia Stepanova  <asiiapine96@gmail.com>
+ *  Distributed under the terms of the GPL v3 license, available in the file LICENSE.
+***/ 
+
+#ifndef SRC_MODULE_CIRCUIT_STATUS_HPP_
+#define SRC_MODULE_CIRCUIT_STATUS_HPP_
+
+#include "periphery/led/led.hpp"
+#include "uavcan/equipment/temperature/Temperature.h"
+#include "periphery/adc/circuit_periphery.hpp"
+#include "uavcan/equipment/power/CircuitStatus.h"
+#include "logger.hpp"
+
+
+class CircuitStatusModule {
+public:
+    static CircuitStatusModule &get_instance();
+    void spin_once();
+
+    float temp = 0;
+    float v5_f = 0.0;
+    float volt = 0;
+    float curr = 0;
```

Comment:
These variables are unused. We don't need to keep them anyway. If you need current or voltage, you can use CircuitPeriphery class directly.